lti/app.py

```diff
@@ -228,10 +229,15 @@ def lti_credentials(request):
     if ( credentials is None ):
       return page_response(lti_credentials_form(request.registry.settings))
 
-    lock = filelock.FileLock("credentials.lock")
-    with lock.acquire(timeout = 1):
-      with open('credentials.txt', 'a') as f:
-        f.write(credentials + '\n')
+    credentials = json.loads(credentials)
+
+    request.db.add(OAuth2UnvalidatedCredentials(
+        client_id=credentials.get("path_a_key"),
+        client_secret=credentials.get("path_a_secret"),
+        authorization_server=credentials.get("path_a_host"),
+        email_address=credentials.get("path_a_email"),
+    ))
```

Comment:
@sheetaluk So as you can see all I need to do to save a new object to the database, is create an `OAuth2UnvalidatedCredentials` object and pass it to `request.db.add()` - that's all there is to it